cylc/flow/network/scan.py

```diff
@@ -94,9 +92,9 @@ def dir_is_flow(listing):
         for path in listing
     }
     return (
-        SERVICE.name in listing
-        or SUITERC.name in listing  # cylc7 flow definition file name
-        or FLOW_FILE in listing  # cylc8 flow definition file name
+        SuiteFiles.SERVICE in listing
```

Comment:
```suggestion
        SERVICE in listing
```
Causing tests to fail.